client/src/components/GridField/GridFieldTable.js

```diff
@@ -16,7 +16,7 @@ class GridFieldTable extends Component {
     }
 
     if (typeof this.props.data !== 'undefined') {
-      // TODO: Generate the header.
+      // noop
     }
```

Comment:
Just remove the whole condition please - it's not doing anything.

Reply:
Updated